assets/script.js

```diff
@@ -0,0 +1,93 @@
+//Declared variables and assigned each to an element within the html document
+var startButton = document.getElementById("start-button");
+var wordDisplay = document.getElementById("word-display");
+var gameMessage = document.getElementById("game-message");
+var timer = document.getElementById("time-left");
+var wins = document.getElementById("wins");
+var losses = document.getElementById("losses");
+
+//Will randomly select a word from the wordArr
+function generateWord(){
+    var wordArr = ["California", "Dragon", "Android"]
+    var randomWord = wordArr[Math.floor(Math.random() * wordArr.length)];
+    console.log(randomWord);
+    wordDisplay.textContent = wordArr[randomWord];

```

Comment:
this does not work. Need to do something like
` wordDisplay.textContent = randomWord;`